code/src/Components/Style.css

```diff
@@ -0,0 +1,202 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+h1 {
+	font-size: 20px;
+	margin-bottom: 30px;
+	font-weight: 400;
+}
+
+h2 {
+	grid-column: 1/4;
+	font-weight: 200;
+	font-size: 18px;
+}
+
+main {
+	width: 90%;
+	max-width: 500px;
+	margin: 30px auto;
+	font-family: menlo;
```

Comment:
I'm not sure that this font is displayed right now, when trying to see how it looks it looks like it's sans serif, but when displayed in the live webpage the font is with serifs 🤔

Reply:
hmm, very strange, when I look in vs code it looks like menlo is not working correctly as you say, but when I look in the browser and untick this font-family it's working. 🤯